Exec/science/wdmerger/tests/tde/inputs

```diff
@@ -37,3 +37,17 @@ amr.plot_per = 10.0
 amr.plot_int = -1
 amr.small_plot_per = 1.0
 amr.small_plot_int = -1
+
+problem.mass_P = 0.60
```

Comment:
since we include the inputs_3d above, we now get everything that used to be in the probin there as problem.xxx parameters here, so if any are different than the defaults and not reset here, then we will be different.  Did you check that?

Reply:
oh... nevermind, we never included a probin here.

Reply:
oh wait.. I guess the way it works is that it still uses the probin in this directory previously, but now it is using what was in the ../../probin, so I think there is still an issue.

Reply:
I did not test this but I did think about it -- the end result should be that we get all of the problem parameters from the main inputs_3d file, followed by the overrides from this file. That results in something slightly different than what we had before only if the default probin parameters in the old probin in the top-level directory didn't match the _prob_parameters defaults, but I'm willing to deal with that at some later point if it ends up being an issue.